newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -138,6 +140,8 @@ export default class AuthenticatedUserProvider extends React.Component<
         onAcceptGameStatsEmail: this._doAcceptGameStatsEmail,
         getAuthorizationHeader: () =>
           this.props.authentication.getAuthorizationHeader(),
+        ownedAssetPackIds: [], // Not using null in order to trigger an update of the assets context.
+        ownedAssetShortHeaders: [], // Not using null in order to trigger an update of the assets context.
```

Comment:
I don't understand those comments, they seem to imply something like `lists trigger updates` but I don't see how

Reply:
They mean that there is an effect listening to this value, and if we go back to null it will not be triggered again, so I'm setting it to an empty array.
But maybe that's not the best strategy